rust/lance-table/src/format.rs

```diff
@@ -41,17 +36,13 @@ pub mod pb {
     #![allow(improper_ctypes)]
     #![allow(clippy::upper_case_acronyms)]
     #![allow(clippy::use_self)]
-    include!(concat!(env!("OUT_DIR"), "/lance.format.pb.rs"));
+    include!(concat!(env!("OUT_DIR"), "/lance.table.rs"));
 }
 
+/// These version/magic values are written at the end of manifest files (e.g. versions/1.version)
 pub const MAJOR_VERSION: i16 = 0;
 pub const MINOR_VERSION: i16 = 1;
 pub const MAGIC: &[u8; 4] = b"LANC";
-pub const INDEX_MAGIC: &[u8; 8] = b"LANC_IDX";
```

Comment:
I removed this.  It was never used anywhere.  I think it's a good idea to a different magic at the end of index files.  But we aren't doing that today.